sqlmesh/core/engine_adapter/base_postgres.py

```diff
@@ -92,13 +92,11 @@ def create_view(
                 **create_kwargs,
             )
 
-    def _get_data_objects(
-        self, schema_name: str, catalog_name: t.Optional[str] = None
-    ) -> t.List[DataObject]:
+    def _get_data_objects(self, schema_name: SchemaName) -> t.List[DataObject]:
         """
         Returns all the data objects that exist in the given schema and optionally catalog.
         """
-        catalog_name = f"'{catalog_name}'" if catalog_name else "NULL"
+        catalog_name = f"'{self.get_current_catalog()}'"
```

Comment:
why don't you have to handle null?

Reply:
Postgres should always have a database that it is operating in and therefore a catalog is always returned.